pymeasure/instruments/keithley/keithley2602B.py

```diff
@@ -80,44 +109,41 @@ def get_trigger_event_description_strings():
 
 
 class Keithley2600DigitalIOPin:
-    def __init__(self, instrument, pin_number):
+    def __init__(self, instrument: Keithley2602B, pin_number: int) -> None:
         self.instrument = instrument
         self.pin_number = pin_number
 
-    def ask(self, cmd):
+    def ask(self, cmd: str) -> Union[str, None]:
         return self.instrument.ask(f"print(digio.trigger[{self.pin_number}].{cmd})")
 
-    def write(self, cmd):
+    def write(self, cmd: str):
         self.instrument.write(f"digio.trigger.[{self.pin_number}].{cmd}")
 
     def check_errors(self):
-        return self.instrument.check_errors()
+        self.instrument.check_errors()
 
     def assert_trigger(self):
         """This method asserts a trigger pulse on one of the digital I/O lines."""
 
         log.info(f"Asserting a trigger pulse on pin number {self.pin_number}.")
         self.write("assert()")
-        self.check_errors()
 
     def clear_trigger(self):
         """This method clears the trigger event detector on a digital I/O line."""
 
         log.info(f"Clearing trigger on pin number {self.pin_number}.")
         self.write("clear()")
-        self.check_errors()
 
-    def get_event_id(self):
+    def get_event_id(self) -> Union[int, None]:
         """This method returns the mode in which the trigger event detector and
         the output trigger generator operate on the given trigger line. See description
         of all the possible EVENT_IDs on page 9-57 of the Series 2600B Reference Manual.
         """
 
         id = self.ask("EVENT_ID")
-        self.check_errors()
         return int(id)
```

Comment:
Along the lines of what @heltluke was mentioning above... None would throw here as int(None) will barf.